src/main/java/de/sventorben/keycloak/authentication/hidpd/HomeIdpDiscoveryAuthenticator.java

```diff
@@ -199,7 +187,7 @@ private static Optional<IdentityProviderModel> discoverHomeIdp(AuthenticationFlo
 
     private static Optional<String> getEmailDomain(UserModel user) {
         Optional<String> domain = Optional.empty();
-        if (user.isEnabled() && user.isEmailVerified()) {
```

Comment:
this probably should be an option

Reply:
Why would you want to redirect already registered users to an IdP based on an unverified email address?

Reply:
Here's an example: User created an account through their Google IdP, their account was upgraded to enterprise and now uses Okta. All company domains should now redirect to Okta and override the users IdP.

Reply:
I don't require users to verify their email, it's postponed as much possible unless there's an action that requires it.